docker-compose.yml

```diff
@@ -1,10 +1,18 @@
 version: '2'
 services:
-  ubuntu:
+  cowrie:
     build:
       context: .
-    image: cowrie:ubuntu
     volumes:
-      - ./cowrie.sysconfig:/etc/default/cowrie:z
+      - /srv/cowrie:/etc/cowrie
```

Comment:
Any reason you prefer /srv over /opt?

Reply:
So this is a little misleading.  These two lines represent 2 different things.  What I'm doing here is:

- removing the need for /etc/default/cowrie.  That was a flat file containing a bunch of ENV variables.  Instead of using that, I changed the container to use real env variables.  That way you just pass the vars in the native docker way, no file needed

- The additional line is just a host mount for /etc/cowrie.  the '/srv' bit there was from my local computer, but it could be any path really.  We still need a persistent volume for /etc/cowrie because it contains the json registration file

Reply:
Ah, ok!

Could we make it `./` instead then?

Reply:
Done!